ui/src/app/app.component.html

```diff
@@ -0,0 +1,74 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-dark">
+  <a class="nav-link" href="#">Payment Simulator</a>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse"
+          aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarCollapse">
+    <ul class="navbar-nav mr-auto">
+      <!--<li class="nav-item">
+        <a class="nav-link" routerLink="/example" routerLinkActive="active">Example</a>
+      </li>-->
+    </ul>
+  </div>
+</nav>
+<div class="container">
+  <br>
+  <br>
+  <br>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="form-group landingLabels">
```

Comment:
You could use the built in bootstrap utilities to align text instead of creating a new class just to right alight. Bootstrap has `.text-right` http://getbootstrap.com/docs/4.1/utilities/text/#text-alignment

Reply:
Who to say we will not have more css in the future? this can be debated right or left In my opinion